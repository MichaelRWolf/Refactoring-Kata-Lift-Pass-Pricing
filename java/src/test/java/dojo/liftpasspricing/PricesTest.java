package dojo.liftpasspricing;

import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Test;

import java.sql.SQLException;
import java.util.Date;
import java.util.List;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class PricesTest {

    @BeforeAll
    public static void createPrices() throws SQLException {
        CostForTypeProvider costForTypeProvider = new CostForTypeProvider() {
            @Override
            public int getCostForLiftTicketType(String liftTicketType) throws SQLException {
                return 17;
            }

            @Override
            public void setLiftPassCostForLiftPassType(int liftPassCost, String liftPassType) throws SQLException {
            }
        };
        HolidaysProvider holidaysProvider = new HolidaysProvider() {
            @Override
            public List<Date> getHolidays() throws SQLException {
                return null;
            }
        };
    }

    @Test
    public void doesSomething() {
        assertEquals(2+2, 4);
    }

    @Test
    public void doPutGetHandlerBusinessLogic() {
        // write some tests for the business logic
    }

}
