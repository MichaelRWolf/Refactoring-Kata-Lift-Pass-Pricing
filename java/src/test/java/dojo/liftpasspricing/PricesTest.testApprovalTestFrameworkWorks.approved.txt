{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 1}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 2}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 3}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 4}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 5}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 6}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 7}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 8}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 9}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 10}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 11}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 12}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 13}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 14}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 15}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 16}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 17}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 18}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 19}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 20}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 21}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 22}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 23}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 24}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 25}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 26}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 27}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 28}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 29}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 30}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 31}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 32}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 33}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 34}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 35}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 36}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 37}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 38}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 39}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 40}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 41}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 42}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 43}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 44}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 45}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 46}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 47}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 48}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 49}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 50}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 51}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 52}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 53}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 54}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 55}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 56}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 57}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 58}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 59}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 60}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 61}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 62}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 63}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 64}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 65}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 66}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 67}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 68}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 69}
{liftTicketType: regular, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 70}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 1}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 2}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 3}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 4}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 5}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 6}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 7}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 8}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 9}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 10}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 11}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 12}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 13}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 14}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 15}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 16}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 17}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 18}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 19}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 20}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 21}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 22}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 23}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 24}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 25}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 26}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 27}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 28}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 29}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 30}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 31}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 32}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 33}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 34}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 35}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 36}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 37}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 38}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 39}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 40}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 41}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 42}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 43}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 44}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 45}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 46}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 47}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 48}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 49}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 50}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 51}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 52}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 53}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 54}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 55}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 56}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 57}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 58}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 59}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 60}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 61}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 62}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 63}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 64}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 65}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 66}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 67}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 68}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 69}
{liftTicketType: night, usageDate: Tue Aug 15 00:00:00 EDT 2023, skierAge: 70}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 1}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 2}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 3}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 4}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 5}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 6}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 7}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 8}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 9}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 10}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 11}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 12}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 13}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 14}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 15}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 16}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 17}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 18}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 19}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 20}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 21}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 22}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 23}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 24}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 25}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 26}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 27}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 28}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 29}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 30}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 31}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 32}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 33}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 34}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 35}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 36}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 37}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 38}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 39}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 40}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 41}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 42}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 43}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 44}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 45}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 46}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 47}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 48}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 49}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 50}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 51}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 52}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 53}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 54}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 55}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 56}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 57}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 58}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 59}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 60}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 61}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 62}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 63}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 64}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 65}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 66}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 67}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 68}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 69}
{liftTicketType: regular, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 70}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 1}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 2}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 3}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 4}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 5}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 6}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 7}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 8}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 9}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 10}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 11}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 12}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 13}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 14}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 15}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 16}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 17}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 18}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 19}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 20}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 21}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 22}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 23}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 24}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 25}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 26}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 27}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 28}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 29}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 30}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 31}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 32}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 33}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 34}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 35}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 36}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 37}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 38}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 39}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 40}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 41}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 42}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 43}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 44}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 45}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 46}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 47}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 48}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 49}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 50}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 51}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 52}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 53}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 54}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 55}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 56}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 57}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 58}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 59}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 60}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 61}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 62}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 63}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 64}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 65}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 66}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 67}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 68}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 69}
{liftTicketType: night, usageDate: Mon Dec 25 00:00:00 EST 2023, skierAge: 70}
